src/main/pages/che-7/end-user-guide/assembly_converting-a-che-6-workspace-to-a-che-7-devfile.adoc

```diff
@@ -19,16 +19,202 @@ summary:
+            "org.eclipse.che.ws-agent",
+            "org.eclipse.che.exec",
+            "org.eclipse.che.terminal"
+          ],
+          "env": {}
+        }
+      },
+      "recipe": {
+        "type": "dockerimage",
+        "content": "quay.io/openshiftio/che-vertx"
+      }
+    }
+  },
+  "projects": [
+    {
+      "links": [],
+      "name": "vertx-http-booster",
+      "attributes": {
+        "language": [
+          "java"
+        ]
+      },
+      "type": "maven",
+      "source": {
+        "location": "https://github.com/openshiftio-vertx-boosters/vertx-http-booster",
+        "type": "git",
+        "parameters": {}
+      },
+      "path": "/vertx-http-booster",
+      "description": "HTTP Vert.x Booster",
+      "problems": [],
+      "mixins": []
+    }
+  ],
+  "name": "wksp-jhwp",
+  "commands": [
+    {
+      "commandLine": "scl enable rh-maven33 'mvn compile vertx:debug -f ${current.project.path} -Dvertx.disableDnsResolver=true'",
+      "name": "debug",
+      "attributes": {
+        "goal": "Debug",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "custom"
+    },
+    {
+      "commandLine": "scl enable rh-maven33 'mvn compile vertx:run -f ${current.project.path} -Dvertx.disableDnsResolver=true'",
+      "name": "run",
+      "attributes": {
+        "goal": "Run",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "custom"
+    },
+    {
+      "commandLine": "scl enable rh-maven33 'mvn clean install -f ${current.project.path}'",
+      "name": "build",
+      "attributes": {
+        "goal": "Build",
+        "previewUrl": ""
+      },
+      "type": "mvn"
+    },
+    {
+      "commandLine": "mvn -Duser.home=${HOME} -f ${CHE_PROJECTS_ROOT}/vertx-http-booster clean package",
+      "name": "vertx-http-booster:build",
+      "attributes": {
+        "goal": "Build",
+        "previewUrl": ""
+      },
+      "type": "mvn"
+    },
+    {
+      "commandLine": "mvn -Duser.home=${HOME} -f ${CHE_PROJECTS_ROOT}/vertx-http-booster vertx:run",
+      "name": "vertx-http-booster:run",
+      "attributes": {
+        "goal": "Run",
+        "previewUrl": "${server.8080/tcp}"
+      },
+      "type": "mvn"
+    }
+  ],
+  "links": []
+}
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+Che 7 devfile
++++ </summary><div> +++
+----
+metadata:
+  name: testing-workspace
+projects:
+  - name: java-web-vertx
+    source:
+      location: 'https://github.com/che-samples/web-java-vertx'
+      type: git
+components:
+  - id: redhat/java/latest
+    type: chePlugin
+  - mountSources: true
+    endpoints:
+      - name: 8080/tcp
+        port: 8080
+    memoryLimit: 512Mi
+    type: dockerimage
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    alias: maven
+    image: 'quay.io/eclipse/che-java8-maven:nightly'
+    env:
+      - value: >-
+          -XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
```

Comment:
I don't think you should add all these env variables here. They are not in the Che 6 workspace config and they make the devfile longer / harder to read.